environments/icds-cas/inventory/citusdb.csv

```diff
@@ -0,0 +1,27 @@
+hostname,host_address,group 1,group 2,group 3,var: hostname
+Cituscoordinator0,100.71.184.115,citusdb,citusdb_cordinator,,MUMGCCWCDPRDCC00
```

Comment:
can we make these lowercase. How about `citusmaster`? Its shorter and easier to type...

Reply:
👍